drivers/mysql/mysql_test.go

```diff
@@ -95,11 +95,20 @@ func (suite *MysqlTestSuite) TestOpen() {
 	})
 
 	suite.T().Run("when connURL is valid and bare uses default configuration", func(t *testing.T) {
-		connectedDriver, teardown := suite.InitializeDriver(testConnURL)
+		connectedDriver, teardown := suite.InitializeDriver(defaultConnURL)
 		defer teardown()
 
+		defaultConfig := getDefaultConfig()
+		cfg := &Config{
+			Config: drivers.Config{
+				MigrationsTable:        defaultConfig.MigrationsTable,
+				StatementTimeoutInSecs: defaultConfig.StatementTimeoutInSecs,
+				MigrationMaxSize:       defaultConfig.MigrationMaxSize,
+			},
+			closeDBonClose: true, // we have created DB from DSN
+		}
 		mysqlDriver := connectedDriver.(*mysql)
-		suite.Assert().EqualValues(defaultConfig, mysqlDriver.config)
+		suite.Assert().EqualValues(cfg, mysqlDriver.config)
```

Comment:
Hmm .. this was `defaultConfig` before. Was this changed because we changed `testConnURL`?

Reply:
Explained above.